src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1244,6 +1245,9 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
             try
             {
+                BoundNode usingVarLoweredNode = LocalUsingVarRewriter.Rewrite(body);
```

Comment:
>usingVarLoweredNode [](start = 26, length = 19)

When can `usingVarLoweredNode` be `null`?

Reply:
Fixed

Reply:
I still see the null check though just one line lower. 

---
In reply to: [203527089](https://github.com/dotnet/roslyn/pull/28587#discussion_r203527089) [](ancestors = 203527089)